lib/jmap/core/filter/operator/logic_filter_operator.dart

```diff
@@ -0,0 +1,19 @@
+import 'package:jmap_dart_client/jmap/core/filter/filter.dart';
+import 'package:jmap_dart_client/jmap/core/filter/filter_operator.dart';
+
+class LogicFilterOperator extends FilterOperator {
+  LogicFilterOperator(Operator operator,Set<Filter> conditions)
```

Comment:
There should be a space between the second parameter and the comma before it.
`LogicFilterOperator(Operator operator, Set<Filter> conditions)`